helm/bloom-vault/README.md

```diff
@@ -0,0 +1,80 @@
+# Kubernetes Helm Chart for Bloom Vault 
+
+Kubernetes deployment for Bloom Vault service (bloom-vault). The deployments, services, ingresses and other resources are installed with Helm charts. Helm allows to install, uninstall, upgrades and rollbacks.
+
+### Requirements
+
+The following are required for this procedure.
+
+#### Environment:
+
+- Kubernetes cluster v1.16 or later
+- Kubernetes config authentication file
+- Helm client installed
+
+  ```
+  brew update
+  brew install helm
+  ```
+
+#### Secrets
+
+The following three secrets are dependencies for bloom-vault and must exist in the same namespace.
+
+- bloom-vault-env-secret: environmental variables
+- bloom-vault-postgres-secret: postgres client certificate
+- myregistrykey: docker registry login
+
+Example secrets are included as yaml files.
+
+
+### Configuration
+
+The application deployment is specified as variables and templates.
+
+Create a new branch for each deployment. Edit the config files for the intended purpose (i.e. dev, stage, prod). For example, the image tag should be specified in the values.yaml.
+
+- Chart.yaml:  
+  Info about chart including chart name, app version and chart version.
+- values.yaml:  
+  The default configuration values for this chart.
+
+### Deploy
```

Comment:
@rcompos thoughts on adding examples that describe the current use state since we're deploying in the `-n stag` and `-n prod` for now?